### ISSUE TITLE
Select max-width option

### ISSUE THREAD

**Issue by dragomano** [sa-si-dev/virtual-select#127]
Hi. Can you add an additional option to change a select max-width?

```
.vscomp-ele {
  display: inline-block;
  width: 100%;
  max-width: 250px; // <= this value
}
```

**Comment by sa-si-dev** [comment]
@dragomano I will add this feature in the future release.

**Comment by sa-si-dev** [comment]
@dragomano This feature has been implemented as `maxWidth` property in the [release v1.0.28](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.28)
